src/RadixDlt.NetworkGateway.PostgresIntegration/CommonDbContext.cs

```diff
@@ -157,6 +159,10 @@ internal abstract class CommonDbContext : DbContext
 
     public DbSet<KeyValueStoreSchemaHistory> KeyValueStoreSchemaHistory => Set<KeyValueStoreSchemaHistory>();
 
+    public DbSet<AccountAuthorizedDepositorHistory> AccountAuthorizedDepositorHistory => Set<AccountAuthorizedDepositorHistory>();
```

Comment:
How about `*EntryHistory`? While this convention is not followed everywhere it aligns with some newer code areas, looks bit more descriptive and aligns partially with the substate naming model we actually operate on?